docs/src/pages/playground.tsx

```diff
@@ -97,6 +97,9 @@ function Homepage() {
     input_5: InputResolution.Resoultion1920x1080,
     input_6: InputResolution.Resoultion1920x1080,
   });
+  const { siteConfig } = useDocusaurusContext();
+  const environment = siteConfig.customFields.environment as string;
```

Comment:
I don't like passing that through the components, this should be a global value. Is there a way to access that field without a hook?

Reply:
As far as I know and have tested, hook is needed, but I'll double-check.